src/progress/pct/compile.p

```diff
@@ -553,31 +564,31 @@ PROCEDURE displayCompileErrors PRIVATE:
   DEFINE INPUT  PARAMETER piColumn  AS INTEGER    NO-UNDO.
   DEFINE INPUT  PARAMETER pcMsg     AS CHARACTER  NO-UNDO.
 
-  DEFINE VARIABLE i AS INTEGER    NO-UNDO .
-  DEFINE VARIABLE c AS CHARACTER  NO-UNDO.
-  DEFINE VARIABLE bit AS INTEGER NO-UNDO.
-  DEFINE VARIABLE memvar AS MEMPTR NO-UNDO.
-  DEFINE VARIABLE include AS LOGICAL NO-UNDO.
+  DEFINE VARIABLE i       AS INTEGER    NO-UNDO .
+  DEFINE VARIABLE c       AS CHARACTER  NO-UNDO.
+  DEFINE VARIABLE bit     AS INTEGER    NO-UNDO.
+  DEFINE VARIABLE memvar  AS MEMPTR     NO-UNDO.
+  DEFINE VARIABLE include AS LOGICAL    NO-UNDO.
 
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Cannot reference "INITIAL" as "init" due to the "require-full-keywords" compiler option. (18494) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=rssw-oe%3Acompiler.warning.18494)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Abbreviated keyword init for INITIAL [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=rssw-oe-main%3Aeu.rssw.antlr.proparse.checks.AbbreviatedKeywords)